src/Generators/MigrationGenerator.php

```diff
@@ -117,4 +121,9 @@ protected function getPath(Model $model, Carbon $timestamp)
     {
         return 'database/migrations/' . $timestamp->format('Y_m_d_His') . '_create_' . $model->tableName() . '_table.php';
     }
+
+    protected function isLaravel7orNewer()
+    {
+        return version_compare(app()->version(), '7.0.0', '>=');
```

Comment:
current laravel version via TestBench `$this->app::VERSION`

Reply:
Yeah, I was trying to see if there was a way to fake it.